extensions/wikia/AdEngine/js/video/uapVideo.js

```diff
@@ -0,0 +1,64 @@
+/* global define */
+define('ext.wikia.adEngine.video.uapVideo', [
+	'ext.wikia.adEngine.adHelper',
+	'ext.wikia.adEngine.domElementTweaker',
+	'ext.wikia.adEngine.video.uapVideoAnimation',
+	'ext.wikia.adEngine.video.videoAdFactory',
+	'wikia.log'
+], function (adHelper, DOMElementTweaker, uapVideoAnimation, videoAdFactory, log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.video.uapVideo';
+
+	function getVideoHeight(width, params) {
+		return width / params.videoAspectRatio;
+	}
+
+	function defaultGetSlotWidth(slot) {
+		return slot.clientWidth;
+	}
+
+	function loadVideoAd(params, adSlot, imageContainer, getSlotWidth) {
+		getSlotWidth = getSlotWidth || defaultGetSlotWidth;
+
+		try {
+			var video = videoAdFactory.create(
+				getSlotWidth(adSlot),
+				getVideoHeight(getSlotWidth(adSlot), params),
```

Comment:
Actually, it is consistent, based on slot width we are are counting video height